Dockerfile.tests

```diff
@@ -0,0 +1,12 @@
+FROM fedora:latest
+
+RUN dnf install -y make python3-pytest python3-devel gcc libattr-devel redhat-rpm-config \
+    && dnf clean all
+
+COPY ./tests /tests
+
+RUN mkdir /src
```

Comment:
The below `COPY . /src` creates the `/src` for you, see end of https://docs.docker.com/engine/reference/builder/#copy